docs/topic-guides/blendable-web-apps/app-shell.md

```diff
@@ -30,8 +30,10 @@ The default content of the app shell is tied to the StarcounterClientFiles versi
 The app shell is invoked automatically for any `Json` with a `Html` property that is returned from your app. The only provision is that your app needs to register the following middleware:
 
 ```csharp
-Application.Current.Use(new HtmlFromJsonProvider());
-Application.Current.Use(new PartialToStandaloneHtmlProvider());
+Starcounter.Application.Current.
+    Use(new HtmlFromJsonProvider());
```

Comment:
You usually place the dot on the new line:

```cs
Starcounter.Application.Current
     .Use(new HtmlFromJsonProvider());
```

Reply:
Done, thanks